modules/ossm-cert-manager-integration-istio.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-security.adoc
+
+:_content-type: CONCEPT
+[id="ossm-cert-manager-integration-istio_{context}"]
+== Integrating Service Mesh with cert-manager and istio-csr
+
+cert-manager adds certificates and certificate issuers as resource types in Kubernetes clusters, and simplifies the process of obtaining, renewing and using those certificates. 
+
+It obtains certificates from a variety of Issuers, both popular public Issuers as well as private Issuers, and ensures the certificates are valid and up-to-date. It also attempts to renew certificates at a configured time before expiry. 
+
+cert-manager enables Istio users to integrate with their private or external PKI. `istio-csr` acts as an external CA for Istio. When it is enabled, applications within a mesh send requests for signing certificates to `istio-csr`. 
+
+.Prerequisites
+
+* cert-manager v1.11+ or community cert-manager operator v1.11+ or cert-manager operator provided by Red Hat v1.10+.
+
+* OSSM operator 2.4+
+
```

Comment:
This might be a good place to call out our support limitations, which need to be noted somewhere here. 

The only cert-manager operator that we support is the recently GAed [cert-manager Operator for Red Hat OpenShift](https://docs.openshift.com/container-platform/4.12/security/cert_manager_operator/cert-manager-operator-install.html) (not sure the official name or version - should check that). If that's viable (did we test against this @jewertow?), we should make sure to use the exact name and link to the installation page. We do NOT provide support istio-csr as of 2.4.  Customers who want support for istio-csr and/or other instances of cert-manager will need to use the services of a vendor like JetStack. We don't have to mention the need for support for another vendor or JetStack specifically (though they did help with this feature), but should at least make our support boundaries very clear.

Reply:
I have added a Note to clarify: "Red Hat provides support for integrating with istio-csr and cert-manager. It does not provide support for the istio-csr or cert-manager components. The use of community cert-manager and istio-csr shown here are for demonstration purposes only."
--

Reply:
Leaving comment open until it is confirmed if the cert-manager integration was tested against [cert-manager Operator for Red Hat OpenShift](https://docs.openshift.com/container-platform/4.12/security/cert_manager_operator/cert-manager-operator-install.html)

Reply:
@jewertow can you confirm if this integration was tested against [cert-manager Operator for Red Hat OpenShift](https://docs.openshift.com/container-platform/4.12/security/cert_manager_operator/cert-manager-operator-install.html)

Reply:
I think we could say that **we test** OSSM with Red Hat cert-manager and cert-manager-istio-csr, **but we don't provide support for these components**, we only provide support for integration with them.
WDYT?

Reply:
To clarify @jewertow and @longmuir , should "OSSM operator 2.4+" be changed to "cert-manager Operator for Red Hat OpenShift"?

I can link to the installation https://docs.openshift.com/container-platform/4.12/security/cert_manager_operator/cert-manager-operator-install.html in an "Additional resources"  section as that is doc style for links.

Reply:
Oh wait...I think I see ... it's not "OSSM operator 2.4+" ... it's the cert-manager v1.10+

That has now been updated to "cert-manager Operator for Red Hat OpenShift" as titled and referenced in the docs.

Reply:
I think we could note that we test this integration with Red Hat cert-manager, but it should work with cert-manager provided by the community operator and helm charts as well.

Reply:
@jewertow The cert-manager docs are a bit confusing there.. it is GA now, the red blob says "If you used the Technology Preview version... you must uninstall it".